pages/common/csvcut.md

```diff
@@ -0,0 +1,20 @@
+# csvcut
+
+> Filter and truncate CSV files. Like unix "cut" command, but for tabular data.
+> Included in csvkit.
+
+- Print indices and names of all columns:
+
+`csvcut -n {{data.csv}}`
+
+- Extract the first and third columns:
+
+`csvcut -c {{1,3}} {{data.csv}}`
+
+- Extract all columns EXCEPT the fourth one:
+
+`csvcut -C {{4}} {{data.csv}}`
+
+- Extract the columns named "id" and "first name" (in that order):
+
+`csvcut -c {{id,"first name"}} {{data.csv}}`
```

Comment:
Not sure, to be honest, and the documentation doesn't clarify. I think in general, if you have columns named with numbers, it's your own dang fault if you end up with confusing behavior. :stuck_out_tongue: